nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/AbstractS3Processor.java

```diff
@@ -82,6 +83,15 @@
             .description("A comma-separated list of Amazon User ID's or E-mail addresses that specifies who should have permissions to change the Access Control List for an object")
             .defaultValue("${s3.permissions.writeacl.users}")
             .build();
+    public static final PropertyDescriptor CANNED_ACL = new PropertyDescriptor.Builder()
+            .name("Canned ACL")
```

Comment:
Would you please use `name()` as a property key ("canned-acl") and use `displayName()` for the UI ("Canned ACL")?  I know you matched the existing code, but we're trying to move to the name+displayName standard.